builders/integrations/indexers/thegraph.md

```diff
@@ -83,7 +83,7 @@ To initialize your subgraph, you'll need to provide some additional information,
 2. Hit enter for **Subgraph slug** to use the default one provided, or change as needed
 3. Again, hit enter for **Directory to create the subgraph in** to use the default one provided, or change as needed
 4. For **Ethereum network**, scroll down and select the Moonbeam network you are working with. Note that the Moonbase Alpha TestNet is labeled as **mbase**
-5. Enter the contract address to index and query data from. The CLI will attempt to fetch the ABI from Moonscan. If it doesn't work, make sure that your contract has been verified and retry if needed. Otherwise, you will need to input it manually as a JSON file after your project has been successfully created
+5. Enter the contract address to index and query data from. The CLI will attempt to fetch the ABI from Moonscan. If it doesn't work, make sure that your contract has been verified and retry if needed. Otherwise, you will need to input it manually as a JSON file after your project has been successfully created. If you don't have a smart contract in mind and want to follow along with the tutorial, you can use the EXR contract address: `0x515e20e6275CEeFe19221FC53e77E38cc32b80Fb`
```

Comment:
Please put the contract address in a code block to give it a copy button for the user. Thanks!